pybabylonjs/args.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 TileDB Inc.
+# Licensed under the MIT License.
+"""Functions to format and check the data and keyword arguments for each data source and visualization mode."""
+
+import os
+import sys
+from .data import *
+
+POINT_CLOUD_KWARG_DEFAULTS = {
```

Comment:
Do we have duplication of the schema validation here, these defaults should be in the schema and for each visualization type the schema should be sub-classed or am I missing something?

Reply:
I removed one, so there is only one now